wallet/src/Pos/Wallet/Web/Swagger/Description.hs

```diff
@@ -52,6 +52,9 @@ instance HasCustomSwagger TestReset where
         "Delete all secret keys. It works in development mode only, \
         \returns HTTP 403 otherwise."
 
+instance HasCustomSwagger TestState where
```

Comment:
This is a philosophical observation, and actually food for though for the upcoming wallet re-design: It feels wrong exposing internal/tests endpoints to the swagger documentation, and it's confusing for 3rd parties to see. We should probably find out a way to still document this (for backend developers) without leaking it to end-users docs.

For now, I think that's fine.